packages/@okta/vuepress-site/docs/guides/quickstart/register-app/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Register your app
+---
+
+In this section, we hook up your Okta org to the app that's going to use it for sign-in. To tell Okta about the app, and to collect the pieces of information you need to integrate into your app, you use the Developer Console's app screen.
+
+For simplicity, in this run-through let's assume that your app is a web app, i.e., a server-side app, and that you want to use the Okta-hosted sign-in page, rather than implement or host your own sign-in page.
+
+To start, find the example applications, provided by Okta on GitHub, for the language or framework you work in:
+
+| Language / Framework | Link                                               |
+|----------------------|----------------------------------------------------|
+| ASP.Net              | <https://github.com/okta/samples-aspnet>           |
+| ASP.NET Core         | <https://github.com/okta/samples-aspnetcore>       |
+| ASP.Net Web Forms    | <https://github.com/okta/samples-aspnet-webforms>  |
+| Java Servlet         | <https://github.com/okta/samples-java-servlet>     |
+| Java Spring          | <https://github.com/okta/samples-java-spring>      |
+| JavaScript Angular   | <https://github.com/okta/samples-js-angular>       |
+| JavaScript React     | <https://github.com/okta/samples-js-react>         |
+| JavaScript Vue       | <https://github.com/okta/samples-js-vue>           |
```

Comment:
I don't think you need the "JavaScript" prefix on these three. Especially since Angular is TypeScript. ;)